app/helpers/ember_rails_helper.rb

```diff
@@ -8,6 +8,11 @@ def render_ember_app(name, &block)
 
     head, body = markup_capturer.capture
 
-    render text: EmberCli[name].index_html(head: head, body: body).html_safe
+    template = EmberCli[name].index_html(head: head, body: body).html_safe
+    begin
+      render text: template
+    rescue ArgumentError
+      render body: template
```

Comment:
Thanks for clarifying.

This project [supports rails>=4.1](https://github.com/thoughtbot/ember-cli-rails/tree/d03f519ed9c2ae2cb413b6bca76ad04da696fd3d#ruby-and-rails-support), so a breaking change in `4.0` is not a show stopper.